georiviere/river/templates/river/river_detail.html

```diff
@@ -0,0 +1,32 @@
+{% extends "mapentity/mapentity_detail.html" %}
+{% load i18n l10n static geojson_tags thumbnail %}
+
+{% block extrabody %}
+    {{ block.super }}
+
+    <script type="text/javascript">
+        $(window).on('detailmap:ready', function (e, data) {
+            var map = data.map;
+
+            //
+            // Source
+            {% if object.source_location %}
+            /*
+                var sourceIcon = L.icon({
+                    iconUrl: "{% static "river/source.png" %}",
+                    iconSize: [{{ view.icon_sizes.source }}, {{ view.icon_sizes.source }}],
+                    iconAnchor: [{{ view.icon_sizes.source }}/2, {{ view.icon_sizes.source }}/2],
+                });*/
```

Comment:
comment to delete ?

Reply:
Even the entire file : it was not supposed to be commited